packages/morbig/morbig.0.9/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+
+maintainer: "Yann Régis-Gianas <yann.regis-gianas@irif.fr>"
+authors: [
+  "Yann Régis-Gianas <yann.regis-gianas@irif.fr>"
+  "Ralf Treinen <ralf.treinen@irif.fr>"
+  "Nicolas Jeannerod <nicolas.jeannerod@irif.fr>"
+]
+license: "GPL3"
+
+homepage: "https://github.com/colis-anr/morbig"
+bug-reports: "https://github.com/colis-anr/morbig/issues"
+dev-repo: "git://github.com/colis-anr/morbig.git"
+
+available: [os != "macos"]
+depends: [
+  "ocaml"                {build & >= "4.03.0"}
```

Comment:
This does not work in opam 1.2, you need to modify availabe as follows:
```
available: [os != "macos" & ocaml-version >= "4.03.0"]
```
or something like that. I can't remember if opam 1.2 wants "osx", "darwin" or "macos", sorry.